components/Search/SearchForm.jsx

```diff
@@ -0,0 +1,60 @@
+import React, { Component } from 'react';
+
+import TextField from 'material-ui/TextField';
```

Comment:
I was so lazy to make this stateless component, because I need to get rid of refs yesterday, let me try.